custom_components/keymaster/coordinator.py

```diff
@@ -526,6 +526,10 @@ async def _handle_lock_state_change(
         if not event:
             return
 
+        # Ignore events without a user_id - they're from zwave - let handle_zwave_js_lock_event handle it
+        if event.context.as_dict()["user_id"] is None:
```

Comment:
Shouldn't this be `event.data.get("user_id")` ?

Reply:
The user_id isn't part of the data member of the event? As far as I can tell, it's only part of the context, which is a separate member.

Reply:
```suggestion
        if event.context.get("user_id") is None:
```
then we can do this instead.

Reply:
I don't believe we can - as far as I can tell, the context object does not implement a Dict interface directly. Which is why I did as_dict() to get a dict out of it. As far as I can tell, it implements basically NO API at all. https://github.com/home-assistant/core/blob/7eb0f2993f25deba1fba9e4dcbcc8d13cf574482/homeassistant/core.py#L1243 - perhaps directly fetching the member would work?